### PR TITLE
add scripts + license

### PR DESCRIPTION
**Pull request by armando-tno**
can you check if I have add the license texts in the scripts correctly?

### CODE REVIEW

File: main_UCARE.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+# This is the main script to generate the Augemented Emission Maps (AEM) from the project uCARe.
+# Please ensure that the necessary packages includded in the requirements.txt are installed correctly
```
Comment (jessica-tno) [inline main_UCARE.py]: Included

File: scripts/graph_output.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#===============================================================================
+
+# Module graph_output.py
+# Module to generate the figures for visual representation of the Augemented Emission Map (AEM)
```
Comment (jessica-tno) [inline scripts/graph_output.py]: Augmented

**Review by jessica-tno — CHANGES_REQUESTED** [review]
Please amend two typos

**Comment by armando-tno** [comment]
typos in main_uCARE.py and graph_output.py are fixed

**Review by jessica-tno — APPROVED** [review]

**Pull request merged**